src/cli/build.ts

```diff
@@ -1,10 +1,27 @@
+import { execSync } from 'child_process';
```

Comment:
I’d suggest an async exec like “execa” which has a really easy interface and keeps it async which is great for stuff like this

Reply:
Went for `execa` in the end - seems very straightforward!